crates/burn-import/src/burn/node/binary.rs

```diff
@@ -230,15 +244,27 @@ impl BinaryNode {
     pub(crate) fn lower(lhs: Type, rhs: Type, output: Type) -> Self {
         let function = match (&lhs, &rhs) {
             (Type::Tensor(_), Type::Tensor(_)) => move |lhs, rhs| quote! { #lhs.lower(#rhs) },
-            _ => panic!("lower is supported for tensor only"),
+            (Type::Tensor(_), Type::Scalar(_)) => move |lhs, rhs| quote! { #lhs.lower_elem(#rhs) },
+            (Type::Scalar(_), Type::Tensor(_)) => {
+                // L < R == R > L
+                move |lhs, rhs| quote! { #rhs.greater_elem(#lhs) }
+            }
+            (lhs, rhs) => panic!("lower is not supported for {lhs:?} > {rhs:?}"),
         };
         Self::new(lhs, rhs, output, BinaryType::Less, Arc::new(function))
     }
 
     pub(crate) fn lower_equal(lhs: Type, rhs: Type, output: Type) -> Self {
         let function = match (&lhs, &rhs) {
             (Type::Tensor(_), Type::Tensor(_)) => move |lhs, rhs| quote! { #lhs.lower_equal(#rhs) },
-            _ => panic!("lower_equal is supported for tensor only"),
+            (Type::Tensor(_), Type::Scalar(_)) => {
+                move |lhs, rhs| quote! { #lhs.lower_equal_elem(#rhs) }
+            }
+            (Type::Scalar(_), Type::Tensor(_)) => {
+                // L <= R == R >= L
+                move |lhs, rhs| quote! { #rhs.greater_equal_elem(#lhs) }
```

Comment:
using `lower_equal_elem` and `greater_equal_elem` instead of  `lower_elem` and `greater_elem`